src/wallet/rpcwallet.cpp

```diff
@@ -4406,8 +4406,15 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
         if (!saplingActive && saplingEntries.size() > 0) {
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, Sapling has not activated");
         }
+        // Do not include Sprout/Sapling notes if using "ANY_SAPLING"/"ANY_SPROUT" respectively
```

Comment:
If the user includes both keywords, all entries will be cleared.  Or is the existence of both keywords caught/handled elsewhere?

Reply:
I changed the logic just below to include `|| (useAnySprout && useAnySapling)`. If the user passes in both values, they will get an error saying that you can't send from both sprout and sapling (I added a test for this too). I would be happy to change this so that we check sooner and the user gets a more specific error message if they pass in both values if we prefer that.